### PR TITLE
Updated Kali Linux Installation and Uninstallation

### PR DESCRIPTION
**Pull request by kiazhi**
Updated the Installation and Uninstallation Bash commands for Kali Linux distro to address the issue #2901.

Tested the steps in Kali Linux on Windows Subsystem for Linux (WSL) on the following version:
```sh
$ sudo cat /etc/os-release
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2018.3"
VERSION_ID="2018.3"
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="https://www.kali.org/"
SUPPORT_URL="https://forums.kali.org/"
BUG_REPORT_URL="https://bugs.kali.org/"
```

```sh
$ pwsh
PowerShell 6.1.0
Copyright (c) Microsoft Corporation. All rights reserved.

https://aka.ms/pscore6-docs
Type 'help' to get help.

PS /> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      6.1.0
PSEdition                      Core
GitCommitId                    6.1.0
OS                             Linux 4.4.0-43-Microsoft #1-Microsoft Wed Dec 31 14:42:53 PST 2014
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

PS /> Get-Content -Path /etc/os-release
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2018.3"
VERSION_ID="2018.3"
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="https://www.kali.org/"
SUPPORT_URL="https://forums.kali.org/"
BUG_REPORT_URL="https://bugs.kali.org/"
```

Version(s) of document impacted
------------------------------
- [x] Impacts 6.next document
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [x] The documented feature was introduced in version (6.x or higher) of PowerShell
- [x] This issue only shows up in version (6.x or higher) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by TravisEz13** [comment]
Did you verify that invoke-webrequest works to an https url after your changes?

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md](https://github.com/kiazhi/PowerShell-Docs/blob/patch-1/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-3.0&branch=pr-en-us-2973) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-4.0&branch=pr-en-us-2973) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.0&branch=pr-en-us-2973) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.1&branch=pr-en-us-2973) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-6&branch=pr-en-us-2973) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C2%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201810022143214180-2973%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gPKDFn7hVB%2BUzMJus174%2B3vg%2FXZWaT2%2BDLZQ6neDW4o%3D&st=2018-10-02T21%3A53%3A29Z&se=2018-11-02T21%3A58%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md [hunk 1]
```diff
@@ -586,12 +586,18 @@ sudo snap remove powershell-preview
 
 ```sh
 # Download & Install prerequisites
-sudo apt-get install libunwind8 libicu55
-wget http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
-sudo dpkg -i libssl1.0.0_1.0.1t-1+deb8u6_amd64.deb
+wget http://ftp.us.debian.org/debian/pool/main/i/icu/libicu57_57.1-9_amd64.deb
+dpkg -i libicu57_57.1-9_amd64.deb
+apt-get install -y curl gnupg apt-transport-https
```
Comment (TravisEz13) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: I had to run `apt-get update` before this command.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md [hunk 2]
```diff
@@ -586,12 +586,18 @@ sudo snap remove powershell-preview
...
-# Install PowerShell
-sudo dpkg -i powershell_6.1.0-1.ubuntu.16.04_amd64.deb
+# Add Microsoft public repository key to APT
+curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+
+# Add Microsoft package repository to the source list
+echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-debian-stretch-prod stretch main" | tee /etc/apt/sources.list.d/powershell.list
+
+# Install PowerShell package
+apt-get install -y powershell
```
Comment (TravisEz13) [inline reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: You have to run `apt-get update` before you can install from the new repository.

**Comment by kiazhi** [comment]
> Did you verify that invoke-webrequest works to an https url after your changes?

Since libssl1.0.x or higher is installed, it does work.

```sh
$ sudo apt search libssl1.*
Sorting... Done
Full Text Search... Done
libssl1.0-dev/kali-rolling 1.0.2o-1 amd64
  Secure Sockets Layer toolkit - development files

libssl1.0.2/kali-rolling,now 1.0.2o-1 amd64 [installed,automatic]
  Secure Sockets Layer toolkit - shared libraries

libssl1.1/kali-rolling,now 1.1.0h-4 amd64 [installed]
  Secure Sockets Layer toolkit - shared libraries
```

Test result using `Invoke-WebRequest` to download powershell package from github releases through HTTPS below. Unless I'm missing something that you may want me to test against? You can give me the commands to run and I will post it back here.

```powershell
PS /> Invoke-WebRequest -Uri "https://github.com/PowerShell/PowerShell/releases/download/v6.1.0/powershell-6.1.0-linux-x64.tar.gz" -OutFile ~/powershell-6.1.0-linux-x64.tar.gz
PS /> Get-ChildItem -Path /home/ryentang/powershell-6.1.0-linux-x64.tar.gz


    Directory: /home/ryentang


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
------          10/3/18  10:55 AM       58271981 powershell-6.1.0-linux-x64.tar.gz


PS /> exit

$ ls -l /home/ryentang/
total 56960
-rw-rw-rw- ryentang ryentang 58271981 Oct  3 10:55 powershell-6.1.0-linux-x64.tar.gz
```

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md (hunk 2 above)
Comment (kiazhi) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: Yes, true. Forgot to include that to update the package list. Good catch.

File: reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md (hunk 1 above)
Comment (kiazhi) [reply reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md]: True, before line 591. I will fix that.

**Comment by kiazhi** [comment]
> BTW, I asked you about libssl before I checked it my self. This is my basic test to verify that it's working correctly. If you get correctly formed json back it's working.
> 
> ```
>  '(Invoke-WebRequest -Uri "https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json").Content'
> ```

@TravisEz13, confirmed it is working.

Result Output below.
```sh
$ sudo cat /etc/os-release
PRETTY_NAME="Kali GNU/Linux Rolling"
NAME="Kali GNU/Linux"
ID=kali
VERSION="2018.3"
VERSION_ID="2018.3"
ID_LIKE=debian
ANSI_COLOR="1;31"
HOME_URL="https://www.kali.org/"
SUPPORT_URL="https://forums.kali.org/"
BUG_REPORT_URL="https://bugs.kali.org/"
```

```sh
$ pwsh
PowerShell 6.1.0
Copyright (c) Microsoft Corporation. All rights reserved.

https://aka.ms/pscore6-docs
Type 'help' to get help.
```

```powershell
PS /> (Invoke-WebRequest -Uri "https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/metadata.json").Content
{
  "StableReleaseTag": "v6.1.0",
  "PreviewReleaseTag": "v6.1.0-rc.1",
  "ServicingReleaseTag": "v6.0.4",
  "ReleaseTag": "v6.1.0",
  "NextReleaseTag": "v6.1.0-rc.1"
}
```

**Comment by TravisEz13** [comment]
No, It can be removed.  I verified the changes.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md](https://github.com/kiazhi/PowerShell-Docs/blob/patch-1/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-3.0&branch=pr-en-us-2973) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-4.0&branch=pr-en-us-2973) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.0&branch=pr-en-us-2973) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-5.1&branch=pr-en-us-2973) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/setup/installing-powershell-core-on-linux?view=powershell-6&branch=pr-en-us-2973) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C3%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201810031827025707-2973%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=EUly0wDFmrt8nJrUDxTYhUcnUERO2ULuGEEpXpk9QCk%3D&st=2018-10-03T18%3A38%3A18Z&se=2018-11-03T18%3A43%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**